_summaries/liu2019large-scale_2.md

```diff
@@ -1,19 +1,90 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
-# Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+paper: liu2019large-scale
+author: Shivi Agarwal
+score: 9
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+
+## The Core Idea of this paper?
+Real world data is long tailed and open-ended distributed. A practical recognition system must classify among majority and minority classes. 
+This paper discusses about the Open Long-Tailed Recognition (OLTR) as learning from naturally distributed data and optimizing the classification 
+accuracy over a balanced test set including head, tail, and open classes. OLTR focuses on imbalanced classification, few-shot learning, and open-set 
+recognition in single integrated algorithm, however the existing classification approaches focus only on one aspect and deliver poorly. The key 
+challenges for OLTR are tail recognition robustness and open-set sensitivity. The core idea of this paper is to develop an integrated OLTR algorithm 
+that maps an image to a feature space such that visual concepts. It can relate to each other based on a learned metric that respects the closed-world 
+classification while acknowledging the novelty of the open world. This dynamic meta-embedding combines a direct image feature and an associated memory 
+feature, with the feature norm indicating the familiarity to known classes. It has modulated attention to encourage the head and tail classes to use 
+different sets of spatial features. As the Meta embedding relates head and tail classes, the modulated attention maintains discrimination between them.
+
+
+## Model and Technical representation:
+
+The OLTR consists of two main modules, dynamic Meta embedding and modulated attention. The former relates and transfers knowledge between head and tail 
+classes and the latter maintains discrimination between them.
```

Comment:
what about the open tail? (never before seen objects). Could be a good idea to briefly mention how these rare cases are handled

Reply:
I have mentioned all the details of implementation I could find in this paper. Couldn't find anything specific to open tail. The rare cases are handled by . Dynamic Meta-Embedding and attention . I have already briefly described these points.